lib/logger.js

```diff
@@ -1,7 +1,57 @@
 'use babel'
 
+import _ from 'lodash'
+
 export default class Logger {
-  error (/* statusCode, result, builder */) {}
-  warning (/* message */) {}
-  info (/* message */) {}
+  constructor () {
+    this.messages = []
+    this._group = null
+    const callback = () => { this.showFilteredMessages() }
+    atom.config.onDidChange('latex.loggingLevel', callback)
```

Comment:
I'm always a fan of making things simpler, so this sounds good to me 👍

---
_Two of my favorite acronyms (and principles) are [KISS](https://en.wikipedia.org/wiki/KISS_principle), and [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)._